packages/typespec-azure-resource-manager/lib/arm.tsp

```diff
@@ -7,16 +7,19 @@ import "@azure-tools/typespec-autorest";
 import "@azure-tools/typespec-azure-core";
 
 import "./arm.foundations.tsp";
-import "./common-types.tsp";
+import "./common-types/common-types.tsp";
+import "./common-types/types.tsp";
+import "./common-types/managed-identity.tsp";
```

Comment:
I really like having the swagger common-types directory structure mirrored here, it will make it easier for us to keep all these types in sync as the swagger common-types evolve